### DISCUSSION TITLE
Add Status Page in a dashboard

### DISCUSSION THREAD

**Discussion by viorelz** [newrelic/nr1-status-pages#187] [Q&A] [answered]
Hi. I created a Status Pages application to pick the RSS from Auth0' status news and I'd like to add this application in a custom dashboard as a widget.

**Comment by aso1124** [comment, marked as answer]
Hi @viorelz - Unfortunately Status Pages cards can't be added as dashboard widgets yet, although we do have this enhancement on our roadmap. It's not yet scheduled for work, but we'll advise when it is.

**Comment by khpeet** [comment]
This now exists as part of the labs widget pack here: https://github.com/newrelic/nr-labs-widget-pack#status-page
